### ISSUE TITLE
More Obvious Notification that New Results are available, and the current results are incomplete

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#229]
sitting in the fishbowl room at the hackathon, many participants were disappointed with the initial results returned from their queries, while it says "calculating" new users didn't understand that more results could/would be returned.

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/cbe1dab7-fdde-4dcb-9285-0dd2b3f72935)

new users didn't see "load new results" 

new users didn't understand that after loading new results, more results could be expected

**Comment by sierra-moxon** [comment]
duplicate of #159, I think?  (closing, of course reopen if I got this wrong).

**Comment by sstemann** [comment]
@sierra-moxon i think we should re-open this on and this is different from #159 , i think #159 is now about speed and/or ability to predict remaining time/results where as this is focused on the UI - and there hasn't been any change on the UI for it being more obvious.

**Comment by dnsmith124** [comment]
I agree @sstemann 

This ticket has actually been on my radar, and I've added a feature that I think should help. I haven't deployed this yet, but I just finished adding a 'sticky' toolbar to the results that remains in the user's view at all times as they scroll the page. It shows the status of the results, and the user can load new results by clicking the prompt that appears there just as they can in the other places where the status is shown. Here's what it looks like: 
![Screen Shot 2023-08-10 at 12 30 51 PM](https://github.com/NCATSTranslator/Feedback/assets/54226761/d4874a8a-390a-46bf-a6e0-dc6e7083a0b4)

It's also collapsable, in case the user doesn't care to see it:
![Screen Shot 2023-08-10 at 12 33 00 PM](https://github.com/NCATSTranslator/Feedback/assets/54226761/cacfce36-3954-4b72-9d5f-2b641b89bc60)

I believe this will help alleviate the issue, since there will now be a reminder of the status of the results visible at all times.

I'll update the ticket once this addition has been deployed

**Comment by Genomewide** [comment]
Can it open when if minimized when there are new results or something changes?

**Comment by sstemann** [comment]
i can't wait to see it!  i assume it will also have the "scrolling" text that Andy has proposed "Querying knowledge graphs, Calculating results,  Reasoning, Processing results, Combining results, Formatting results" instead of "Calculating"? i worry it could be a bit long now?

**Comment by dnsmith124** [comment]
@Genomewide I like the idea that we're reminding users that new results have arrived even when they've closed it, but I'm hesitant to re-expand it unprompted after a user has explicitly clicked to close it. I think there's middle ground here with a chime of some sort or a bubble that appears to show there's a new interaction available. Can you add this to the list of things we could use design and user testing for, and get it on @desiyopp 's radar?

@sstemann Once I get the final, approved verbiage for the scrolling text it will be updated to use that as well. Though it will likely only feature the verb from each phrase ("Querying", "Calculating", etc) since you're right, the full phrases would take up too much space. 

Is this an acceptable way to integrate that new verbiage?

**Comment by dnsmith124** [comment]
@sstemann Nevermind, a small increase in the width of the toolbar (+10px) allows it to support the full phrase. 

I've implemented the new scrolling text in both the original loading screen before results are displayed and in the places where the loading status is communicated to the user (above and below the results, in the sticky toolbar).

Waiting on it to be deployed before closing.

**Comment by sstemann** [comment]
in prod as of 9/6
